dhis-2/dhis-api/src/main/java/org/hisp/dhis/icon/IconStore.java

```diff
@@ -71,12 +70,9 @@ public interface IconStore {
   List<Icon> getIcons(IconQueryParams params);
 
   /**
-   * Check if the store contains entries for all provided keys
-   *
-   * @param keys a set of keys
-   * @return true, if all exist
+   * @return all existing keys
```

Comment:
```suggestion
   * @return all existing keys (including variants)
```

I am constantly thinking about if what I am looking at is the normalized key or the key of a variant. Not sure if someone else has the same difficulty 😅